code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -42,81 +43,113 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	var/obj/item/stock_parts/cell/cell = null
 	var/obj/machinery/camera/camera = null
 
-	// Components are basically robot organs.
+	/// Components are basically robot organs.
 	var/list/components = list()
 
-	var/obj/item/robot_parts/robot_suit/robot_suit = null //Used for deconstruction to remember what the borg was constructed out of..
+	/// Used for deconstruction to remember what the robot was constructed out of.
+	var/obj/item/robot_parts/robot_suit/robot_suit = null
+	/// Used for deconstruction to remember what the robot was constructed out of.
 	var/obj/item/mmi/mmi = null
 
 	var/obj/item/pda/silicon/robot/rbPDA = null
 
 	var/datum/wires/robot/wires = null
 
+	/// Is the robot's maintenance panel open?
 	var/opened = FALSE
 	var/custom_panel = null
 	var/list/custom_panel_names = list("Cricket")
+	/// Robot skins with multiple variants for different modules. They require special handling to make their eyes display.
 	var/list/custom_eye_names = list("Cricket", "Standard")
 	var/emagged = 0
```

Comment:
```suggestion
	var/emagged = FALSE
```
Fun fact, this is the last place in the codebase that uses `0` instead of `FALSE` for emagged!

Reply:
Such power.